src/emulator/modules/SceGxm/SceGxm.cpp

```diff
@@ -380,13 +380,13 @@ EXPORT(int, sceGxmDraw, SceGxmContext *context, SceGxmPrimitiveType primType, Sc
     assert(indexData != nullptr);
 
     if (!host.gxm.isInScene) {
-        return SCE_GXM_ERROR_NOT_WITHIN_SCENE;
+        return error("sceGxmDraw", SCE_GXM_ERROR_NOT_WITHIN_SCENE);
```

Comment:
I have fixed all those with `__func__` in my Fixes PR, after rebasing on Rin's PR that was merged to master.

Reply:
So idk you can merge it but I'm gonna change it in my PR anyway.